riak_test/yz_rt.erl

```diff
@@ -325,31 +351,41 @@ search_expect(solr, {Host, Port}, Index, Name0, Term0, Shards, Expect)
     Name = quote_unicode(Name0),
     Term = quote_unicode(Term0),
     URL = internal_solr_url(Host, Port, Index, Name, Term, Shards),
-    lager:info("Run search ~s", [URL]),
+    lager:info("Expect search ~s", [URL]),
```

Comment:
Oh yeah, that change was mostly for my own debugging purposes. There were two separate but identical lager calls that said "Run search" and I couldn't tell which was which so I changed one of them to "Expect search" instead.